docs/docs/recipes.md

```diff
@@ -853,6 +853,129 @@ export const pageQuery = graphql`
 - [Guide to creating pages from data programmatically](/docs/programmatically-create-pages-from-data/)
 - [Example repo](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown) for this recipe
 
+### Sourcing data from Contentful
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start/)
+- A [Contentful account](https://www.contentful.com/)
+- The [Contentful CLI](https://www.npmjs.com/package/contentful-cli) installed
+
+#### Directions
+
+1. Log in to Contentful with the CLI and follow the steps. It will help you create an account if you don't have one.
+
+```shell
+contentful login
+```
+
+2. Create a new space. Make sure to save the space ID given to you at the end of the command.
```

Comment:
```suggestion
2. Create a new space if you don't already have one. Make sure to save the space ID given to you at the end of the command. If you already have a Contentful space and space ID, you can skip this step.

Note: for new accounts, you can overwrite the default onboarding space. Check to see the [spaces included with your account](https://app.contentful.com/account/profile/space_memberships).
```

How about this, @muescha?

Reply:
using the exisiting one can be a different template and not the blog template and then the other code in this receipe not works.

maybe just mention here that they have 2 free spaces...

or only go on when it is a blog template. but if we use a existing space, then it gets tricky because the user need to find out where he find the tokens.